core-bundle/src/Cron/Cron.php

```diff
@@ -34,8 +37,38 @@ class Cron
      * @param \Closure():CronJobRepository      $repository
      * @param \Closure():EntityManagerInterface $entityManager
      */
-    public function __construct(private \Closure $repository, private \Closure $entityManager, private LoggerInterface|null $logger = null)
+    public function __construct(
+        private \Closure $repository,
+        private \Closure $entityManager,
+        private CacheItemPoolInterface $cachePool,
+        private LoggerInterface|null $logger = null,
+    ) {
+    }
+
+    public function hasMinutelyCliCron(): bool
+    {
+        return $this->cachePool->getItem(self::MINUTELY_CACHE_KEY)->isHit();
+    }
+
+    public function updateMinutelyCliCron(string $scope): PromiseInterface|null
     {
+        if (self::SCOPE_CLI !== $scope) {
+            return null;
+        }
```

Comment:
This does not (yet) work as expected, I created an issue for that: https://github.com/contao/contao/issues/5645